package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
     "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "test": "NODE_ENV=test npm run seed && jest",
```

Comment:
running jest will automatically set the NODE_ENV to test so we shouldn't have to do this explicitly :)

Reply:
ok. understood- will change this Thank you